js/controller.js

```diff
@@ -0,0 +1,118 @@
+goog.module('wordsearch.controller');
+
+const model = goog.require('wordsearch.model');
+const view = goog.require('wordsearch.view');
+
+class Controller {
+  /** @param {!model.GameBoard} wordSearchBoard */
```

Comment:
Both `@param` comments should be in the same JSDoc block.

```
  /** 
   * @param {!model.GameBoard} wordSearchBoard
   * @param {!view.View} myView
   */
```

Reply:
Updated.